src/websocket.js

```diff
@@ -105,19 +121,19 @@ class WebSocket extends EventTarget {
   }
 
   set onopen(listener) {
-    this.addEventListener('open', listener);
```

Comment:
Instead of adding `replaceEventListener` which will pollute the prototype chain we can just modify all of the setters to be this:

```js
set onopen(listener) {
  delete this.listeners.open;
  this.addEventListener('open', listener);
}
```

Reply:
ok 👌